anvil/src/udev.rs

```diff
@@ -523,7 +523,9 @@ impl DrmLeaseHandler for AnvilState<UdevData> {
                 builder.add_connector(conn);
                 builder.add_crtc(*crtc);
                 let planes = backend.drm.planes(crtc).map_err(LeaseRejected::with_cause)?;
-                builder.add_plane(planes.primary.handle);
+                for plane in &planes.primary {
```

Comment:
@Drakulix I am not sure if this is correct?

Reply:
No it isn't. All resources passed here, are given to the client and I don't think any client has use for multiple primary planes. We should just pick one here (ideally a free one :sweat_smile: ).

Reply:
I somewhat expected this to be incorrect :smile: 

Just picking a currently free one might again result in duplicate plane usage. I added the claim logic to the
lease builder. Afaict this should be good?

Reply:
Looks correct now :+1: